src/attpc_engine/kinematics/excitation.py

```diff
@@ -24,18 +27,38 @@ def sample(self, rng: Generator) -> float:
         float
             The sampled value
         """
-        ...
+        pass
+
+    def sample_theta(self, rng: Generator) -> float:
```

Comment:
I think we need to think about this a little bit. In principle the Excitation distribution has nothing to do with the polar angle. We can make a new distribution protocol (PolarDistribution) for the polar angle, etc.